core/templates/pages/contributor-dashboard-page/translation-language-selector/review-translation-language-selector.component.ts

```diff
@@ -89,15 +89,6 @@ export class ReviewTranslationLanguageSelectorComponent implements OnInit {
             this.activeLanguageCode];
         }
       });
-
-    this.contributionOpportunitiesBackendApiService
-      .getPreferredTranslationLanguageAsync()
-      .then((preferredLanguageCode: string|null) => {
-        if (preferredLanguageCode) {
-          this.populateLanguageSelection(
-            preferredLanguageCode);
-        }
-      });
```

Comment:
@sagangwee `this.populateLanguageSelection()` will call `/getreviewableopportunitieshandle`. This caused a duplicated query when switching to the My Contributions tab.

Reply:
Hm I would think we would still want to select the user's preferred language if they have one. Maybe we should just not populate the language selection if the language is already set instead of removing this altogether.

Reply:
I agree. Done.